tests/test_save.py

```diff
@@ -33,14 +33,15 @@ def run_simulation(save=False, dx=0.01, dy=0.01, a=0.5, timesteps=100):
     a, h, s = symbols('a h s')
     eqn = Eq(u.dt, a * (u.dx2 + u.dy2))
     stencil = solve(eqn, u.forward)[0]
-    op = Operator(stencils=Eq(u.forward, stencil), subs={a: 0.5, h: dx, s: dt},
-                  nt=timesteps, shape=(nx, ny), spc_border=1, time_order=1)
+    op = StencilKernel(stencils=Eq(u.forward, stencil), subs={a: 0.5, h: dx, s: dt},
```

Comment:
Just a sidenote, but why are we flipping this test in this PR? Should this not have been done in a separate one?